_posts/2019-01-17-image-service-deprecation.md

```diff
@@ -10,7 +10,7 @@ categories:
 tags: []
 ---
 
-The AGRC brownout of the mapserv image services was very successful. We want to thank you for your participation and apologize for any inconveniences. The few people that were affected and caught by surprise are now informed and are on the migration path forward.
+The AGRC brownout of the mapserv image services on mapserv.utah.gov from January 5 through 10, 2019, in preparation for the ArcGIS Server upgrade was very successful. We want to thank you for your participation and apologize for any inconvenience that the brownout may have caused. The few people that were negatively affected and were caught by surprise are now informed about the deprecation of image services and are on the migration path forward.
 
 With the brownout behind us, AGRC has chosen the date of **Thursday, February 14**, to permanently remove the image services. The deprecation Utah Change Management System reference number is `CHG0019495` and will be discussed at the next two change management meetings.
```

Comment:
Also, so we say: "If more time is needed for your migration, please let Matt Peters from AGRC at MPeters@utah.gov know so we can address them as soon as possible."--I think we should revise to "If more time is needed for your migration, please let Matt Peters from AGRC at mpeters@utah.gov know so we can make other arrangements as soon as possible." 

Right now we have a subject/verb agreement with "if more time is needed...so we can address them..." And also, this is more of a general note, could we maybe consider changing the template contact language? I just don't think it reads all that smoothly for some reason. Like, maybe we could switch to something like "...please let Matt Peters (mpeters@utah.gov) from AGRC know..." ? I think it's just the "from" and then "at" that throws me off a little bit.